File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseRoutesLocation.scala

package org.openapitools.client.model


case class ResponseRoutesLocation (
    _id: String,
    _properties: List[ResponseRoutesProperties]
)
object ResponseRoutesLocation {
    def toStringBody(var_id: Object, var_properties: Object) =
        s"""
        | {
        | "id":$var_id,"properties":$var_properties
        | }
        """.stripMargin
}

File: fsharp-functions/OpenAPI/src/model/ResponseRoutesLocation.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.ResponseRoutesProperties

module ResponseRoutesLocation = 

  //#region ResponseRoutesLocation

  [<CLIMutable>]
  type ResponseRoutesLocation = {
    [<JsonProperty(PropertyName = "id")>]
    Id : string;
    [<JsonProperty(PropertyName = "properties")>]
    Properties : ResponseRoutesProperties[];
  }
  
  //#endregion
  